### DISCUSSION TITLE
Options Trades

### DISCUSSION THREAD

**Discussion by rohitpaul** [ghostfolio/ghostfolio#969] [Ideas]
Would love to see options trades being tracked.

**Comment by mostlyvirtual** [comment]
Are there any plans to implement this?

**Reply by dtslvr** [reply]
Hi @mostlyvirtual

There are no concrete plans for option trades at the moment. I lack the expertise to develop it.

You are welcome to look into it if you are interested. It seems like the required data is available from [Yahoo Finance](https://github.com/gadicc/node-yahoo-finance2/blob/devel/docs/modules/options.md).

**Reply by mostlyvirtual** [reply]
I appreciate your reply. 

Even manual entry without live/active tracking would be a great feature.
i.e. 
On 2022-12-10 sold a PUT expiring 2023-01-17 at the 50 strike price for $230 in premium.
On 2023-01-10 bought it back for $110 so a $120 profit, or paid $300 so lost $70 on the trade.

If the PUT expired worthless on 2023-01-17, then I would enter $0, so the total profit was $230.
If the PUT was in the money and I was forced to buy the 100 shares, then I would still enter $0, since the PUT made the profit, but now I bought the 100 shares at the 50 strike price and they might be worth $40 each, so I'm making an unrealized loss of $10/share, but that can be considered a separate transaction, just like I would buy 100 shares without any options involved.

The same goes for selling CALL options.

---

On the opposide side of the fence, buying PUTs or CALLs is pretty much the same, but in reverse. I would love to go through the logic with you.

---

Of course, if you're open to add that functionality, you could also use the Yahoo Finance data, since it's trivial at that point.

Later on there could be added functionality for people wanting to do certain strategies, so they may account for those, but I think the above is fantastic for most people

**Reply by frwalzer** [reply]
Any news here? I really would need Options support too. Generally the same as stock tracking just that you need to construct a correct identifier from option data (base stock, expiration date, type and option price). Initially could be entered manually I assume. And of course it requires to sell options to start with. And then buy back later or let expire worthless on option date. Buying Options shoUld work in same way as stocks but need to consider exporation too. Option strategies typically consist on a combination of CALL and PUT options. So this can be done by two entries.
